src/App/Listener/CreatePostFromMessage.php

```diff
@@ -0,0 +1,123 @@
+<?php
+namespace Ushahidi\App\Listener;
+
+use Ushahidi\Core\Entity\PostRepository;
+use Ushahidi\Core\Entity\Message;
+use Ushahidi\Core\Entity\MessageRepository;
+use Ushahidi\Core\Entity\TargetedSurveyStateRepository;
+
+class CreatePostFromMessage
```

Comment:
This is where creating new posts from messages and any field mapping happens now